### DISCUSSION TITLE
Caching results with strapi-plugin-rest-cache

### DISCUSSION THREAD

**Discussion by niddu85** [VirtusLab/strapi-plugin-navigation#279] [Q&A]
Hello,
I'm trying to cache results from the navigation plugin using strapi-plugin-rest-cache but I'm not able to properly configure it since the cache plugin needs the content type name to cache (eg api::button.button )
Is there anyone that already configured it properly?

**Comment by Johannes-Schiel** [comment]
i have the same problem did you find something?

**Comment by niddu85** [comment]
unfortunately not yet

**Comment by cyp3rius** [comment]
@Kronos66 do you have an idea how it might be done? As I know with one of our commercial projects there is REST API cache.

**Reply by Johannes-Schiel** [reply]
I would love to hear that :)
